content/docs/user-guide/what-is-dvc.md

```diff
@@ -1,16 +1,17 @@
 # What Is DVC?
 
-**Data Version Control** is a new type of data versioning, workflow and
+**Data Version Control** is a new type of data versioning, workflow, and
 experiment management software, that builds upon [Git](https://git-scm.com/)
 (although it can work stand-alone). DVC reduces the gap between established
 engineering tool sets and data science needs, allowing users to take advantage
 of new [features](#core-features) while reusing existing skills and intuition.
 
 ![](/img/reproducibility.png) _DVC codifies data and ML experiments_
 
-Data science experiment sharing and collaboration can be done through a regular
-Git flow (commits, branching, pull requests, etc.), the same way it works for
-software engineers.
+The same way that software engineers use Git, data scientists can use DVC to
+apply a regular flow to project sharing and collaboration (commits, branching,
+pull requests, etc.). Using Git and DVC, data science and machine learning teams
+can version experiments, manage large datasets, and make projects reproducible.
```

Comment:
I like it! You could even put flow and ML closer like this:
```suggestion
apply a regular flow to machine learning project collaboration and sharing (commits, branching,
pull requests, etc.). Using Git and DVC, data science teams
can version large datasets, manage experiments, and make projects reproducible.
```
> Needs formatting

Notice I also reordered some list items in a way I think makes a little more sense. But in terms of the right terms, looks pretty complete. Maybe even too much for a single paragraph haha.

Reply:
Wait no... We still changed the meaning of the paragraph 🤔 

Let's keep the topic about _data science experiment collaboration and sharing_ please. If we need to talk about _regular flow to machine learning projects_ separately, let's use a separate sentence/paragraph. Thanks

Reply:
A bit confused with the feedback here. 

My initial version had "...a regular Git flow to ML project...". Your initial feedback was that changed the meaning, so I removed the "ML" there and expanded the paragraph to use the complete "machine learning" term and additional features (an additional intro to the features below). 

So my latest doesn't have "regular flow to machine learning projects" — it is in a separate sentence/paragraph. The added sentence is better in SEO terms than my original version with "ML + flow", so I left it out on purpose.

Reply:
Yeah sorry I went back and forth on this one...

Reply:
> Your initial feedback was that changed the meaning

Yes but it didn't refer to where/how to insert the "ML" term or to the extra info added (that's all good), but rather to avoid switching the subject from "data science experiments" to "flow to machine learning projects".

Reply:
> separate sentence/paragraph. The added sentence is better in SEO terms

That's OK. I'm talking about the change in the 1st sentence.

Reply:
My mistake. I get you now... I wasn't looking at the original version. Updated to retain the original 1st sentence and add the new 2nd sentence.

Reply:
Great. Thanks!